core/test/client/unit/models/role_test.js

```diff
@@ -0,0 +1,22 @@
+import {
+    describeModel,
+    it
+} from 'ember-mocha';
+
+describeModel('role', function () {
+    it('provides a lowercase version of the name', function () {
+        var model = this.subject({
+            name: 'Author'
+        });
+
+        expect(model.get('name')).to.equal('Author');
+        expect(model.get('lowerCaseName')).to.equal('author');
+
+        Ember.run(function () {
```

Comment:
Anything that has an async side effect (like a binding) has to be run in an `Ember.run()` when Ember is set to testing mode.

So, since `role.name` is bound to `lowerCaseName` by way of a computed property, it has to go inside `Ember.run()`.